.circleci/config.yml

```diff
@@ -55,103 +28,214 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/target/inspection-results
 
-  no-merge-commits:
+  validate-with-maven-script:
```

Comment:
Nearly all jobs fall into one of two categories: we run `mvn...` from a bash script or run a bash script. So, I have parameterized two jobs named this way. You are able to supply the script name and image you want to run it on, then the name of the job is used as the argument to the script, which is also the name that appears in the CI view in the PR.